c10/util/llvmMathExtras.h

```diff
@@ -147,7 +147,7 @@ std::size_t countTrailingZeros(T Val, ZeroBehavior ZB = ZB_Width) {
 namespace detail {
 template <typename T, std::size_t SizeOfT>
 struct LeadingZerosCounter {
-  static std::size_t count(T Val, ZeroBehavior) {
+  constexpr static std::size_t count(T Val, ZeroBehavior) {
```

Comment:
I think these utilities are from LLVM and we should probably try to either keep it in sync with upstream or make a change log so we can update it later.

Reply:
I prefer adding a change log.